http-gateway/service/httpApi.go

```diff
@@ -145,6 +150,29 @@ func resourceLinksMatcher(r *http.Request, rm *router.RouteMatch) bool {
 	return false
 }
 
+func eventsMatcher(r *http.Request, rm *router.RouteMatch) bool {
+	paths := splitURIPath(r.RequestURI, uri.Events)
+	// api/v1/events/?timestamp={timestamp}
+	return len(paths) == 1 && strings.HasPrefix(paths[0], "?"+uri.TimestampQueryKey+"=")
+}
```

Comment:
I think it can be removed, if we use default handler

Reply:
Removed